### PR TITLE
feat(twitch): twitch docs

### PR DESCRIPTION
**Pull request by mxgic1337**
## 🔧 What does this fix? 🔧

- Themed [Twitch docs](https://dev.twitch.tv/docs/).
2024-11-11-174443_hyprshot

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

**Comment by isabelroses** [comment]
hey @mxgic1337, thanks for so many PRs, would you like to become a maintainer of the twitch userstyle. If your happy to do so take a look at our [docs for adding yourself as a maintainer](https://github.com/catppuccin/userstyles/blob/main/docs/userstylesyml.md#adding-yourself-as-a-maintainer)

File: styles/twitch/catppuccin.user.css [hunk 1]
```diff
@@ -711,6 +711,382 @@
   }
 }
 
+@-moz-document url("https://dev.twitch.tv/"), url-prefix("https://dev.twitch.tv/docs"), url-prefix("https://dev.twitch.tv/code"), url-prefix("https://dev.twitch.tv/products"), url-prefix("https://dev.twitch.tv/showcase"), url-prefix("https://dev.twitch.tv/support"), url-prefix("https://discuss.dev.twitch.com/embed/topics")
```
Comment (uncenter) [inline styles/twitch/catppuccin.user.css:714]: Is there a reason we need to do all these prefixes?
```suggestion
@-moz-document domain("dev.twitch.tv")
```

File: styles/twitch/catppuccin.user.css [hunk 2]
```diff
@@ -711,6 +711,382 @@
...
+      .nav__links a {
+        color: @text;
+      }
+      .nav__links a:hover {
+        color: @subtext0;
+      }
+      .nav__links a.active {
+        color: @accent-color;
+        border-color: @accent-color;
+      }
```
Comment (uncenter) [inline styles/twitch/catppuccin.user.css:804]:
```suggestion
      .nav__links a {
        color: @text;
        
        &:hover {
          color: @subtext0;
        }
        &.active {
          color: @accent-color;
          border-color: @accent-color;
        }
      }
```

File: styles/twitch/catppuccin.user.css [hunk 3]
```diff
@@ -711,6 +711,382 @@
...
+        color: @text;
+      }
+      input::placeholder {
+        color: @subtext0;
+      }
+      input:focus {
+        background: @crust !important;
+        box-shadow: 0 0 3px 1px fade(@accent-color, 70%);
+        color: @text;
+      }
```
Comment (uncenter) [inline styles/twitch/catppuccin.user.css:1017]:
```suggestion
      input {
        color: @text;
        &::placeholder {
          color: @subtext0;
        }
        &:focus {
          background: @crust !important;
          box-shadow: 0 0 3px 1px fade(@accent-color, 70%);
          color: @text;
        }
      }
```
Does `input::placeholder` need to be set here? Our base styles should cover it.

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by uncenter** [comment]
Oop my bad.

**Comment by uncenter** [comment]
Alright, all fixed. Pushed some changes to syntax highlighting as well.

**Comment by uncenter** [comment]
Just these tag buttons on https://dev.twitch.tv/code/ is all that I can find unthemed:

CleanShot 2024-11-11 at 13 18 32

**Review by uncenter — APPROVED** [review]

**Pull request merged**